.circleci/config.yml

```diff
@@ -122,7 +122,7 @@ references:
     environment:
       PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
     name: Install dependencies
-    command: yarn install --frozen-lockfile
+    command: yarn install --immutable
```

Comment:
waaat does v2 work on the calypso repo already???!!! :mindblown:

Reply:
oooooops that's v2 thing isn't it. i guess it's just getting ignored now.

Reply:
went back to the right flag

Reply:
Are we not going to use v2 up front?? :D